.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Comment:
I see. Just stating pull_request allows it to run test when anybody creates PR. Didn't know that before.